### PR TITLE
added to app deployment stuff about pre/post deployment

### PR DESCRIPTION
**Pull request by mikecroft**

### CODE REVIEW

**Comment by mikecroft** [comment]
Is this enough? Any more needed?

File: documentation/extended-documentation/app-deployment/app-deployment.adoc [hunk 1]
```diff
@@ -1,11 +1,46 @@
...
++
+*Aim*: passing a list of commands as a file to the server to run post-boot.
+This is where commands such as service configuration commands should be run.
+
+[NOTE]
+====
+There are some important caveats to note:
+
+* Both files should be saved in a text format, with each command on it's own line.
+* Lines starting with a '#' are treated as comments
```
Comment (OndrejM) [inline documentation/extended-documentation/app-deployment/app-deployment.adoc]: A line is also ignored when there is a blank space before #, and also empty and blank lines are ignored.
Lines with an erroneous command are also ignored, with an Exception in the logs.

**Comment by mikecroft** [comment]
@OndrejM I created an internal ticket for the help as well :smile: it might even be number 1708! It's definitely an important one, though, since there are one or two other places the help needs an improvement or fix

File: documentation/extended-documentation/app-deployment/app-deployment.adoc (hunk 1 above)
Comment (mikecroft) [inline documentation/extended-documentation/app-deployment/app-deployment.adoc]: Would you reword that line to be:

> Lines where the first non-whitespace character is `#` are treated as comments

Comment (OndrejM) [inline documentation/extended-documentation/app-deployment/app-deployment.adoc]: Makes sense. But I would add another sentence:
"The files can include blank lines which will be ignored."

**Comment by mikecroft** [comment]
Added a couple of extra lines @OndrejM

**Pull request merged**